httpx/_models.py

```diff
@@ -958,10 +968,17 @@ def iter_raw(self) -> typing.Iterator[bytes]:
 
         self.is_stream_consumed = True
         self._num_bytes_downloaded = 0
+        chunker = ByteChunker(chunk_size=chunk_size)
+
         with map_exceptions(HTTPCORE_EXC_MAP, request=self._request):
-            for part in self._raw_stream:
-                self._num_bytes_downloaded += len(part)
-                yield part
+            for raw_stream_bytes in self._raw_stream:
+                self._num_bytes_downloaded += len(raw_stream_bytes)
+                for chunk in chunker.decode(raw_stream_bytes):
```

Comment:
What do you think about

```python3
yield from chunker.decode(raw_stream_bytes)
# ...
yield from chunker.flush()
```
(I know this approach is not applicable for `async` copy of this function, but may be we can use it  there?)